vnext/ReactUWP/Modules/AlertModuleUwp.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "AlertModuleUwp.h"
+#include <winrt/Windows.UI.Xaml.Controls.h>
+
+#include <Utils\ValueUtils.h>
+
+#pragma warning(push)
+#pragma warning(disable : 4146)
+#include <cxxreact/JsArgumentHelpers.h>
+#pragma warning(pop)
+namespace winrt {
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+} // namespace winrt
+
+#if _MSC_VER <= 1913
+// VC 19 (2015-2017.6) cannot optimize co_await/cppwinrt usage
+#pragma optimize("", off)
+#endif
+
+namespace react {
+namespace uwp {
+//
+// Alert
+//
+
+const char *AlertModule::name = "RCTAlert";
+
+AlertModule::~AlertModule() = default;
+
+std::string AlertModule::getName() {
+  return name;
+}
+
+std::map<std::string, folly::dynamic> AlertModule::getConstants() {
+  return {};
+}
+
+std::vector<facebook::xplat::module::CxxModule::Method>
+AlertModule::getMethods() {
+  return {
+      Method(
+          "showAlert",
+          [](folly::dynamic args, Callback actionResult) {
+            AlertModule::showAlert(args, actionResult);
+          }),
+
+  };
+}
+
+/*static*/ winrt::fire_and_forget AlertModule::showAlert(folly::dynamic &args, Callback actionResult) {
+
+  std::wstring alertTitle;
+  std::wstring alertContent;
+  std::wstring alertPBT;
+  std::wstring alertSBT;
+  std::wstring alertCBT;
```

Comment:
You could probably be a little more verbose with the naming here for clarify, especially since they each have easily understandable and specific titles (e.g. 'positive' button, gives people a good sense of what the button is supposed to do and is also consistent with RN documentation).